README.md

```diff
@@ -87,7 +87,7 @@ Once all the resources are created, you will need to get the connection informat
 1. Navigate to **Overview** and copy _Server name_ and _Server admin login name_
 1. Open the [Azure Cloud Shell](https://shell.azure.com?WT.mc_id=javascript-38807-aapowell) and log into the `psql` cli:
 
-   - `psql --host <server> --user <username> --port=5432 -p`
```

Comment:
It appears we don’t need the `-p` here as we are already specifying the port, it complains and it says it's already specified.

Reply:
I'm having issues with this step as per Slack conversation, however in my many attempts I also got a SSL message where it says a SSL connection is required. Not sure if the above command allows SSL connections.

<img width="1103" alt="Pasted Graphic 6" src="https://user-images.githubusercontent.com/737821/137425772-8f7b6e33-5e97-49b4-8f01-c794d0eaaf25.png">

Do we need to disable this or pass in another option for SSL?

I tried the `psql "host=xxx dbname=xxx sslmode=require"` style command as well with no luck either.

<img width="909" alt="Pasted Graphic 7" src="https://user-images.githubusercontent.com/737821/137425817-efbe6f8f-7e71-40a2-9c92-b7b74412617c.png">

Reply:
I thought that `-p` was password, let me check that flag 😅

Reply:
Not in my testing, I've not needed to enable that